### ISSUE TITLE
Delete RAR contents

### ISSUE THREAD

**Issue by papaxris** [SickRage/SickRage#3361]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master / 66babc76d28646807c1040a3e225072b410489ad

OS: Linux-4.4.0-64-generic-x86_64-with-Ubuntu-16.04-xenial

What you did: Auto Postprocess with an "Unpack Directory" and "Delete RAR contents" enabled

What happened: Sickrage performed all the expected procedures with the exception of removing the unpack contents (contents left in unpack directory after performing all postprocess procedures).  As if the option "Delete RAR contents" was bypassed.

What you expected:  SickRage to remove the unpack contents from the unpack directory once the postprocessing has copied the contents into its respective library.  How can I help bring this functionality back?

Logs:
```
Non error. No Logs.
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by miigotu** [comment]
This is a known issue. We are working on it.
#3358

**Comment by papaxris** [comment]
Thanks for providing the update with the link.  I wanted to do my part to help improve the project.

**Comment by TATUMTOT** [comment]
Yes i have been having this as well it started back when the RAR stuff was adjusted. I'm on Windows.

**Comment by papaxris** [comment]
Hey Tatumtot,

That's good info.  I am using a Linux version of SR.

**Comment by miigotu** [comment]
The PR is marinating. For now it is sort of working, things just arent being deleted properly. We need some time to rest on that because of how much chaos there was and how much frustration we built up trying to rush and fix it. I went a week with I bet less than 10 hours total sleep trying to fix it to get it back to how it is.

We will continue on the PR when we feel ok with it again.
